rpm/template.spec.em

```diff
@@ -1,3 +1,7 @@
+@{
+# Connext is not available for RHEL - ignore any deps related to it
+BuildDepends = [d for d in BuildDepends if not 'connext' in d]
```

Comment:
I just peeked upstream and rosidl_typesupport_fastrtps packages aren't listed there but do you know why? Is it a leftover oversigh? Are the individual typesupport packages not needed at build time after all? Or is there something that distinguishes rosidl_typesupport_fastrtps from rosidl_typesupport_connext in this regard?

Reply:
I truly have no idea.

Reply:
But it's a really good question.
Should I exclude 'fastrtps' here as well in case it shows up at some point?

Reply:
Updated to avoid FastRTPS as well in 31a6bdb.